### ISSUE TITLE
Private Tab Text Unreadable

### ISSUE THREAD

**Issue by jonathansampson** [brave/browser-laptop#7715]
- Did you search for similar issues before submitting this one?
Yes

- Describe the issue you encountered:
Upon opening a new Private Tab, the text in the tab is hard to read due to lack of contrast.

- Platform (Win7, 8, 10? macOS? Linux distro?):
Desktop

- Brave Version (revision SHA):
0.14.0

- Steps to reproduce:
    1. Open a new tab
    2. Note the text in the tab is hard to read
    
- Actual result:
Tab text has very little contrast.

- Expected result:
Text should have more contrast.

- Will the steps above reproduce in a fresh profile? If not what other info can be added?
Yes

- Is this an issue in the currently released version?
Yes

- Can this issue be consistently reproduced?
Yes

- Screenshot if needed:
![image](https://cloud.githubusercontent.com/assets/815158/23925513/f4068e2c-08e5-11e7-90b4-e6af28b1c5f7.png)

- Any related issues:

**Comment by NejcZdovc** [comment]
I think this should be fixed in 0.14.0

**Comment by bsclifton** [comment]
yikes!

@NejcZdovc good call- +1 on that

**Comment by luixxiul** [comment]
`release-notes/exclude` as the issue does not exist on 0.13 series
